src/linkml_transformer/transformer/object_transformer.py

```diff
@@ -2,11 +2,12 @@
 from dataclasses import dataclass
 from typing import Any, Dict, Optional, Type, Union
 
+from asteval import Interpreter
```

Comment:
Had a look at various different packages for "safely" evaluating strings as python. This one seemed fairly reasonable in allowing a range of useful operations. Obviously evaluating user input is always going to be dodgy, and [it's particularly bad in python](https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html). Anyway, I wanted something that would allow me to demonstrate the kinds of transform that I will need to do whilst making it slightly harder for potential hackers who are keenly monitoring this repo to take over everyone's computers.